Panic after TradePropose

```
panic: runtime error: index out of range [6] with length 6

goroutine 24322 [running]:
github.com/tdex-network/tdex-daemon/pkg/wallet.(*Wallet).BlindSwapTransactionWithData(0xc011a91c20, 0xc011c99000, 0x4b64, 0xc011c765d0, 0xc055d7b950, 0x0, 0x2c, 0x0, 0x0, 0xc011bebdd0)
	/tdex-daemon/pkg/wallet/blind.go:373 +0x7fb
github.com/tdex-network/tdex-daemon/internal/core/application.fillProposal(0xc011b02c00, 0x18, 0x18, 0x1109e80, 0xc0109646c0, 0xc010f42a80, 0x33, 0x33, 0xc011b0b4a0, 0xf, ...)
	/tdex-daemon/internal/core/application/trade_service.go:648 +0xc05
github.com/tdex-network/tdex-daemon/internal/core/application.tradeManager.FillProposal(...)
	/tdex-daemon/internal/core/application/types.go:193
github.com/tdex-network/tdex-daemon/internal/core/application.(*tradeService).TradePropose(0xc0109f93e0, 0x1102de0, 0xc005d02690, 0xc01094b040, 0x40, 0xc01094b080, 0x40, 0x0, 0x1109e80, 0xc0109646c0, ...)
	/tdex-daemon/internal/core/application/trade_service.go:287 +0x775
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.tradePropose(0x11050e0, 0xc0109f93e0, 0xc0516c07c0, 0x1108f20, 0xc005ef8820, 0xe20501, 0xc005ef8820)
	/tdex-daemon/internal/interfaces/grpc/handler/trade.go:213 +0x1cd
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.TradePropose(...)
	/tdex-daemon/internal/interfaces/grpc/handler/trade.go:64
github.com/tdex-network/tdex-protobuf/generated/go/trade._Trade_TradePropose_Handler(0xe3e880, 0xc005daac40, 0x1106940, 0xc0000ca3c0, 0x1, 0xdf9be0)
	/go/pkg/mod/github.com/tdex-network/tdex-protobuf@v0.0.0-20210324125236-78f6c6c0618b/generated/go/trade/trade_grpc.pb.go:282 +0x10b
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/interceptor.streamLogger(0xe3e880, 0xc005daac40, 0x1106940, 0xc0000ca3c0, 0xc005e8ec80, 0x1025700, 0x1102de0, 0xc005d02690)
	/tdex-daemon/internal/interfaces/grpc/interceptor/logger.go:27 +0xbe
google.golang.org/grpc.(*Server).processStreamingRPC(0xc0000c8700, 0x110a840, 0xc0107e9d40, 0xc010dbe800, 0xc010a39110, 0x1596880, 0x0, 0x0, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1448 +0x522
google.golang.org/grpc.(*Server).handleStream(0xc0000c8700, 0x110a840, 0xc0107e9d40, 0xc010dbe800, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1521 +0xc9c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc053dd6010, 0xc0000c8700, 0x110a840, 0xc0107e9d40, 0xc010dbe800)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xa5
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x1fd

```

Stll present in the latest tagged image

```
panic: runtime error: index out of range [3] with length 3

goroutine 1526 [running]:
github.com/tdex-network/tdex-daemon/pkg/wallet.(*Wallet).BlindSwapTransactionWithData(0xc011daf860, 0xc01226b500, 0x31f0, 0xc01224a3f0, 0xc01195d500, 0x0, 0x2c, 0x2, 0x0, 0xc012161dd0)
	/tdex-daemon/pkg/wallet/blind.go:373 +0x7fb
github.com/tdex-network/tdex-daemon/internal/core/application.fillProposal(0xc011ddbc80, 0x18, 0x18, 0x110b380, 0xc011931710, 0xc011ddbe00, 0x18, 0x18, 0xc0107f8c80, 0x14, ...)
	/tdex-daemon/internal/core/application/trade_service.go:650 +0xc05
github.com/tdex-network/tdex-daemon/internal/core/application.tradeManager.FillProposal(...)
	/tdex-daemon/internal/core/application/types.go:208
github.com/tdex-network/tdex-daemon/internal/core/application.(*tradeService).TradePropose(0xc010442c00, 0x11042e0, 0xc011683590, 0xc0118d3c00, 0x40, 0xc0118d3c40, 0x40, 0x0, 0x110b380, 0xc011931710, ...)
	/tdex-daemon/internal/core/application/trade_service.go:287 +0x775
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.tradePropose(0x11065e0, 0xc010442c00, 0xc011688a40, 0x110a420, 0xc0119800c0, 0xe21801, 0xc0119800c0)
	/tdex-daemon/internal/interfaces/grpc/handler/trade.go:213 +0x1cd
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/handler.traderHandler.TradePropose(...)
	/tdex-daemon/internal/interfaces/grpc/handler/trade.go:64
github.com/tdex-network/tdex-protobuf/generated/go/trade._Trade_TradePropose_Handler(0xe3fb80, 0xc01396c050, 0x1107e40, 0xc013950c00, 0x1, 0xdfaee0)
	/go/pkg/mod/github.com/tdex-network/tdex-protobuf@v0.0.0-20210324125236-78f6c6c0618b/generated/go/trade/trade_grpc.pb.go:282 +0x10b
github.com/tdex-network/tdex-daemon/internal/interfaces/grpc/interceptor.streamLogger(0xe3fb80, 0xc01396c050, 0x1107e40, 0xc013950c00, 0xc01169c3c0, 0x1026ad0, 0x11042e0, 0xc011683590)
	/tdex-daemon/internal/interfaces/grpc/interceptor/logger.go:27 +0xbe
google.golang.org/grpc.(*Server).processStreamingRPC(0xc0102e96c0, 0x110bd40, 0xc011666fc0, 0xc013907c00, 0xc013968300, 0x1598880, 0x0, 0x0, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1448 +0x522
google.golang.org/grpc.(*Server).handleStream(0xc0102e96c0, 0x110bd40, 0xc011666fc0, 0xc013907c00, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1521 +0xc9c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc011675180, 0xc0102e96c0, 0x110bd40, 0xc011666fc0, 0xc013907c00)
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xa5
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x1fd

```

--- a/pkg/wallet/blind.go
+++ b/pkg/wallet/blind.go
@@ -161,11 +161,14 @@ func (w *Wallet) BlindSwapTransactionWithKeys(opts BlindSwapTransactionWithKeysO
 
 	ptx, _ := pset.NewPsetFromBase64(opts.PsetBase64)
 
-	inKeysLen := len(opts.InputBlindingKeys)
-	inBlindingKeys := make([]pset.BlindingDataLike, inKeysLen, inKeysLen)
-	for i, in := range ptx.Inputs {
+	inKeysLen := len(ptx.Inputs)
+	inBlindingKeys := make([]pset.BlindingDataLike, 0, inKeysLen)
+	for _, in := range ptx.Inputs {
 		script := hex.EncodeToString(in.WitnessUtxo.Script)
-		inBlindingKeys[i] = pset.PrivateBlindingKey(opts.InputBlindingKeys[script])
+		inBlindingKeys = append(
+			inBlindingKeys,
+			pset.PrivateBlindingKey(opts.InputBlindingKeys[script]),
+		)
 	}
 
 	outBlindingKeys := make(map[int][]byte)
@@ -366,11 +369,14 @@ func (w *Wallet) BlindSwapTransactionWithData(opts BlindSwapTransactionWithDataO
 
 	ptx, _ := pset.NewPsetFromBase64(opts.PsetBase64)
 
-	dataLen := len(opts.InputBlindingData)
-	inBlindingData := make([]pset.BlindingDataLike, dataLen, dataLen)
-	for i, in := range ptx.Inputs {
+	dataLen := len(ptx.Inputs)
+	inBlindingData := make([]pset.BlindingDataLike, 0, dataLen)
+	for _, in := range ptx.Inputs {
 		script := hex.EncodeToString(in.WitnessUtxo.Script)
-		inBlindingData[i] = opts.InputBlindingData[script].ToBlindingData()
+		inBlindingData = append(
+			inBlindingData,
+			opts.InputBlindingData[script].ToBlindingData(),
+		)
 	}
 
 	outBlindingKeys := make(map[int][]byte)

